kolena/_experimental/quality_standard.py

```diff
@@ -91,3 +98,58 @@ def download_quality_standard_result(
         for model in models:
             result_dfs.append(_download_quality_standard_result(dataset, [model], metric_groups, intersect_results))
         return pd.concat(result_dfs, axis=1)
+
+
+@dataclass(frozen=True)
+class CopyQualityStandardRequest:
+    dataset_id: int
+    source_dataset_id: int
+    include_metric_groups: bool = True
+    include_test_cases: bool = True
```

Comment:
I think this should be under https://github.com/kolenaIO/kolena/blob/trunk/kolena/_api/v2/quality_standard.py